app/src/components/portal/Header.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div>
+    <md-app-toolbar class="md-dense md-primary" id="reset_bg">
+      <div class="md-toolbar-row viz-u-postion__rel">
+        <div class="md-toolbar-section-start">
+          <md-button class="md-icon-button" @click="toggler">
+            <md-icon>menu</md-icon>
+          </md-button>
+
+          <router-link to="/" class="header-logo">
+            <span class="md-title"><img id="logo" src="@/assets/img/materialsmine_logo_sm.png"></span>
+          </router-link>
+        </div>
+        <div class="md-toolbar-section-end header_nav">
+          <div class="nav nav_menu u--inline">
+            <ul>
+              <li><router-link to="/nm" v-slot="{ navigate, href }" custom>
+                <a :href="href" @click="navigate">NanoMine</a>
+              </router-link></li>
+              <li><router-link to="/mm" v-slot="{ navigate, href }" custom>
+                <a :href="href" @click="navigate">Metamine</a>
```

Comment:
Should this be MetaMine to match NanoMine? or should it be Nanomine to match Metamine

Reply:
No should be separate